nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/EncodeContent.java

```diff
@@ -237,4 +297,14 @@ public void process(final InputStream in, final OutputStream out) throws IOExcep
             out.flush();
         }
     }
+
+    private static EncodingType getEncodingTypeFromString(final String encodingTypeValue) {
```

Comment:
No need to add suffix `FromString` as it is obvious from the argument.
```suggestion
    private static EncodingType getEncodingType(final String encodingTypeValue) {
```